engine/src/org/pentaho/di/trans/steps/scriptvalues_mod/ScriptValuesAddedFunctions.java

```diff
@@ -2641,12 +2666,12 @@ public static Boolean isMailValid( Context actualContext, Scriptable actualObjec
           return Boolean.FALSE;
         }
 
-        retval = Boolean.TRUE;
+        isValid = Boolean.TRUE;
 
       } catch ( Exception e ) {
         throw Context.reportRuntimeError( "Error in isMailValid function: " + e.getMessage() );
       }
-      return retval;
+      return isValid;
```

Comment:
It looks like isValid can potentially be null. Is this acceptable for code that calls this?